index.d.ts

```diff
@@ -0,0 +1,85 @@
+type ValueOf<T> = T[keyof T];
+
+export type Dtypes = {
+    "<u1": {
+        name: "uint8";
+        size: 8;
+        arrayConstructor: typeof Uint8Array;
+    };
+    "|u1": {
+        name: "uint8";
+        size: 8;
+        arrayConstructor: typeof Uint8Array;
+    };
+    "<u2": {
+        name: "uint16";
+        size: 16;
+        arrayConstructor: typeof Uint16Array;
+    };
+    "|i1": {
+        name: "int8";
+        size: 8;
+        arrayConstructor: typeof Int8Array;
+    };
+    "<i2": {
+        name: "int16";
+        size: 16;
+        arrayConstructor: typeof Int16Array;
+    };
+    "<u4": {
+        name: "uint32";
+        size: 32;
+        arrayConstructor: typeof Int32Array;
+    };
+    "<i4": {
+        name: "int32";
+        size: 32;
+        arrayConstructor: typeof Int32Array;
+    };
+    "<u8": {
+        name: "uint64";
+        size: 64;
+        arrayConstructor: typeof BigUint64Array;
+    };
+    "<i8": {
+        name: "int64";
+        size: 64;
+        arrayConstructor: typeof BigInt64Array;
+    };
+    "<f4": {
+        name: "float32";
+        size: 32;
+        arrayConstructor: typeof Float32Array;
+    };
+    "<f8": {
+        name: "float64";
+        size: 64;
+        arrayConstructor: typeof Float64Array;
+    };
+};
+
+export type Parsed = ValueOf<{
```

Comment:
This is return type of `parse` method.

It is tagged union. So you can narrow the type using if or switch statements.

![image](https://github.com/aplbrain/npyjs/assets/63112453/ec107ba0-4c68-4daf-9d65-eba8457fc6ea)